Test/CoreSDK.Test/Shared/Extensibility/MetricManagerTest.cs

```diff
@@ -150,8 +150,8 @@ public void AggregatedMetricTelemetryIntervalDurationPropertyIsPositiveInteger()
             Assert.Equal(1, aggregatedMetric.Count);
             Assert.Equal(1, aggregatedMetric.Properties.Count);
 
-            Assert.True(aggregatedMetric.Properties.ContainsKey("IntervalDurationMs"));
-            Assert.True(long.Parse(aggregatedMetric.Properties["IntervalDurationMs"]) > 0);
+            Assert.True(aggregatedMetric.Properties.ContainsKey("Microsoft.ApplicationInsights.Metrics.Aggregation.IntervalMs"));
+            Assert.True(long.Parse(aggregatedMetric.Properties["Microsoft.ApplicationInsights.Metrics.Aggregation.IntervalMs"]) > 0);
```

Comment:
That's good, but to Dmitry's point below, are they going to increase the cost of transmission and billing to the customer?